environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - azure-storage-blob
   - boto
   - boto3
+  - mypy-boto3-s3
```

Comment:
This should be part of the pre-commit hooks here: https://github.com/data-engineering-collective/minimalkv/blob/20a516329605cbe0c776975cc1ee01a97f8c7dc0/.pre-commit-config.yaml#L29